live_sentiment.py

```diff
@@ -0,0 +1,63 @@
+import sys
+
+import torch
+from torch.autograd import Variable
+import numpy as np
+
+import models
+import utils
+import settings
+from colored import fg, bg, stylize 
+
+def get_live_sentiment(model, feature):
+    """Takes a trained model and a list of features and returns the
+    estimated scores for each timestep"""
+
+    # Inference
+    feature = Variable(feature.permute(1, 0, 2))
+    out = model(feature)
+
+    return(out)
+
+def rating_to_color(rating):
+    """Takes a rating from 0 to 5 and converts that to a grey scale x-term
+    safe color"""
+
+    val = int(rating/5. * 23 + 232)
+    val = max(0, min(255, val))
+    return val
+
+if __name__ == "__main__":
+    # load model
+    model = models.SimpleLSTM(settings.HIDDEN_SIZE)
```

Comment:
Now we are starting to hardcode the model in lots of places. Perhaps it is time to add a "CURRENT_MODEL" in settings.

Reply:
yeah, thats true, should be done as a separate PR though right?